src/Domain/ContentRenderer/CodeRenderer.php

```diff
@@ -47,4 +48,22 @@ private function getLanguageClasses(): array {
 		return [ 'language-' . htmlspecialchars( $this->language ) ];
 	}
 
+	/**
+	 * @return (string|string[])[]
+	 */
+	private function getWrapperAttributes( string $contentUrl ): array {
+		$attributes = [];
+
+		$attributes['class'] = $this->getWrapperClasses();
+
+		$attributes['data-toolbar-order'] = 'copy-to-clipboard';
+
+		if ( $this->showEditButton == true ) {
+			$attributes['data-toolbar-order'] = 'bitbucket-edit,' . $attributes['data-toolbar-order'];
+			$attributes['data-src'] = $contentUrl;
```

Comment:
We should have a new test in CodeRendererTest to make sure these 2 attributes are set correctly.